### PR TITLE
new packagae and class commited by qa1

### PR DESCRIPTION
**Pull request by yoganandbheema23**
tl create review the code once

### CODE REVIEW

File: src/test/java/patientpac/patientlogin.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package patientpac;
+
+public class patientlogin {
+
+	public static void main(String[] args) {
```
Comment (testingpoint09) [inline src/test/java/patientpac/patientlogin.java:5]: add few line qateam

**Review by testingpoint09 — CHANGES_REQUESTED** [review]
add approriate code changes in the file .

File: src/test/java/patientpac/patientlogin.java (hunk 1 above)
Comment (testingpoint09) [reply src/test/java/patientpac/patientlogin.java:5]: good and please merge the code team.

**Review by testingpoint09 — APPROVED** [review]
PR is apporved

**Comment by yoganandbheema23** [comment]
patient page added

**Pull request merged**